designer/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/definition/package.scala

```diff
@@ -135,6 +141,14 @@ package object definition {
     def apply(action: CustomAction): UICustomAction = UICustomAction(
       name = action.name,
       allowedStateStatusNames = action.allowedStateStatusNames,
+      displayPolicy = action.displayPolicy match {
+        case Some(displayPolicy) =>
+          displayPolicy match {
+            case deployment.CurrentlyViewedProcessVersionIsDeployed =>
+              Some(UICustomActionDisplayPolicy.CurrentlyViewedProcessVersionIsDeployed)
+          }
+        case None => None
+      },
```

Comment:
WDYT about changing this to simple `flatMap`?

Reply:
I think `map` even would be better!